doc/manual/contracts.md

```diff
@@ -38,7 +38,7 @@ Here, `x` is bound to a `Number` contract. When evaluating `x`, the following st
 are performed:
 
 1. Evaluate `1 + 1`.
-2. Check that the result is a number.
+2. Check that the result is a `Number`.
```

Comment:
Hmm, while your suggestion is equally valid, I don't think the original text is incorrect (or less correct) either. I think it's supposed to be a sentence in plain English, using the common word "number" on purpose.

While the difference is small, IMHO it's a bit less of a circular definition to say that

> `Number` check that something is a number

rather than

> `Number` checks that something is a `Number`

Typically, the contract was `Num` before 1.0, in which case the second wording would a bit more obscure.

```suggestion
2. Check that the result is a number.
```

Reply:
makes sense